config/environment.js

```diff
@@ -6,6 +6,11 @@ module.exports = function(environment) {
     environment: environment,
     baseURL: '/',
     locationType: 'auto',
+    googleMap: {
+      lazyLoad: true,
+      apiKey: 'AIzaSyCoc_T6TNxu5Mnw28aJleIm63fR7WzViUU'
```

Comment:
I believe we'll need to get a key for the production domain as well, otherwise google maps will pop up an alert when this is deployed